scripts/MIRfix.py

```diff
@@ -3188,29 +3195,29 @@ def correct(corid,flanking,countcorrected,countcorrectedTonew,listofnew,listofne
                 correctmir=str(listoldstatus[corrind+3])
                 correctmirstar=str(listoldstatus[corrind+4])
                 orien=str(listoldstatus[corrind+5])
-                coor1=int(longseq.index(correctmir))
-                coor2=int(longseq.index(correctmirstar))
+                coor1=int(longseq.find(correctmir))
```

Comment:
here index should be fine as we have to find the mir, if its missing we want an error